projects/1.21/assets/psi/psi/lang/zh_cn.json

```
@@ -0,0 +1,1123 @@
+  "psi.spellpiece.selector_block_broken.desc": "获取工具所破坏的方块",
+  "psi.spellpiece.selector_block_side_broken": "选择符：方块破坏面",
+  "psi.spellpiece.selector_block_side_broken.desc": "获取工具破坏方块时选定的面",
+  "psi.spellpiece.selector_attack_target": "选择符：攻击目标",
+  "psi.spellpiece.selector_attack_target.desc": "获取施术者所攻击的实体",
+  "psi.spellpiece.selector_sneak_status": "选择符：潜行状态",
+  "psi.spellpiece.selector_sneak_status.desc": "获取施术者在潜行与否（未潜行为1，潜行时为0）",
+  "psi.spellpiece.selector_time": "选择符：时间",
+  "psi.spellpiece.selector_time.desc": "获取CAD的内部时间（以刻计）",
+  "psi.spellpiece.selector_attacker": "选择符：攻击者",
+  "psi.spellpiece.selector_attacker.desc": "获取攻击施术者的实体",
+  "psi.spellpiece.selector_damage_taken": "选择符：所受伤害",
+  "psi.spellpiece.selector_damage_taken.desc": "获取施术者受到的伤害量",
+  "psi.spellpiece.selector_ruler_vector": "选择符：矢量尺",
+  "psi.spellpiece.selector_ruler_vector.desc": "获取矢量尺中设定的矢量",
+  "psi.spellpiece.selector_item_presence": "选择符：物品实在性",
+  "psi.spellpiece.selector_item_presence.desc": "获取物品栏中物品组的物品数量",
+  "psi.spellpiece.selector_block_presence": "选择符：方块实在性",
+  "psi.spellpiece.selector_block_presence.desc": "获取方块的实在性和固体性",
+  "psi.spellpiece.selector_saved_vector": "选择符：所存矢量",
+  "psi.spellpiece.selector_saved_vector.desc": "获取CAD内存中存储的一个矢量",
+  "psi.spellpiece.selector_eidos_changelog": "选择符：情报体变更记录",
+  "psi.spellpiece.selector_eidos_changelog.desc": "获取情报体变更记录中的一个位置",
+  "psi.spellpiece.selector_nearby_charges": "选择符：遥控弹药计数",
+  "psi.spellpiece.selector_nearby_charges.desc": "获取指定位置附近遥控型子弹的数量",
+  "psi.spellpiece.selector_nearby_players": "选择符：附近玩家",
+  "psi.spellpiece.selector_nearby_players.desc": "获取指定位置附近的玩家",
+  "psi.spellpiece.selector_nearby_vehicles": "选择符：附近可骑乘",
+  "psi.spellpiece.selector_nearby_vehicles.desc": "获取指定位置附近的可骑乘实体",
+  "psi.spellpiece.selector_nearby_falling_blocks": "选择符：附近下落方块",
+  "psi.spellpiece.selector_nearby_falling_blocks.desc": "获取指定位置附近的下落方块",
+  "psi.spellpiece.selector_nearby_glowing": "选择符：发光",
+  "psi.spellpiece.selector_nearby_glowing.desc": "获取指定位置附近的发光和荧光实体",
+  "psi.spellpiece.selector_caster_energy": "选择符：施术者能量",
+  "psi.spellpiece.selector_caster_energy.desc": "获取施术者的Psi能量水平",
+  "psi.spellpiece.selector_caster_battery": "选择符：施术者电池",
+  "psi.spellpiece.selector_caster_battery.desc": "获取施术者的CAD电池能量水平",
+  "psi.spellpiece.selector_sucession_counter": "选择符：成功计数",
+  "psi.spellpiece.selector_sucession_counter.desc": "获取该盔甲部件成功施术的次数",
+  "psi.spellpiece.selector_is_elytra_flying": "选择符：装备鞘翅",
+  "psi.spellpiece.selector_is_elytra_flying.desc": "如果施术者装备了鞘翅，返回1",
+  "psi.spellpiece.selector_transmission": "选择符：最新消息",
+  "psi.spellpiece.selector_transmission.desc": "获取CAD在指定频道上获取的最新消息",
+  "psi.spellpiece.selector_item_count": "选择符：物品计数",
+  "psi.spellpiece.selector_item_count.desc": "获取你物品栏中与目标槽位中物品一致的物品数",
+  "psi.spellpiece.selector_tick_time": "选择符：读刻时间",
+  "psi.spellpiece.selector_tick_time.desc": "获取焦点维度读刻时间的均值",
+  "psi.spellpiece.selector_tps": "选择符：每秒刻数",

```

```suggestion
  "psi.spellpiece.selector_tps": "选择符：TPS",
```
TPS大概是常用术语了？
只是个人想法，具体看你